src/components/CourtCard.js

```diff
@@ -0,0 +1,83 @@
+import { FaStar } from "react-icons/fa";
+import BasketballIcon from "../images/BasketballIcon.svg";
+import LocationIcon from "../images/LocationIcon.svg";
+import OpeningIcon from "../images/OpeningIcon.svg";
+import { Image } from "cloudinary-react";
+
+function CourtCard({ court, index, onClick }) {
+  console.log(index);
+  return (
+    <article className="listCourtCards" onClick={onClick} key={index}>
+      <section className="listWhiteBackground">
+        <img
+          src={BasketballIcon}
+          className="listBasketballIcon icons"
+          alt="BasketballIcon"
+        />
```

Comment:
```suggestion
        <img
          src={BasketballIcon}
          className="listBasketballIcon icons"
          alt="BasketballIcon"
          width=200
          height=200
        />
```
For sure you have to set your own values :) just put some for demonstration 

"Use both width and height to set the intrinsic size of the image, allowing it to take up space before it loads, to mitigate content layout shifts."

**width**
The intrinsic width of the image in pixels. Must be an integer without a unit.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img